README.md

```diff
@@ -1,32 +1,38 @@
-# php-coding-standards
-Our coding standards for PHP applications.
+# PHP Coding Standards
 
-## Setup
+At [Jump24](https://jump24.co.uk/) we pride ourselves on keeping our coding standards under tight control, this is why we built this package.
 
-Add the following to your `composer.json` file.
-```
-"repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/JumpTwentyFour/php-coding-standards"
-        }
-    ],
+## Installation
+
+To install this package, simply use compose:
+
+```bash
```

Comment:
Think might be typo here of double composer require.

Reply:
Good eye!